src/library.js

```diff
@@ -1917,13 +1917,16 @@ LibraryManager.library = {
   time: function(ptr) {
     var ret = (Date.now()/1000)|0;
     if (ptr) {
-      {{{ makeSetValue('ptr', 0, 'ret', 'i32') }}};
+      {{{ makeSetValue('ptr', 0, 'ret', 'i64') }}};
     }
     return ret;
   },
 
-  difftime: function(time1, time0) {
-    return time1 - time0;
+  difftime: function(time1lo, time1hi, time0lo, time0hi) {
```

Comment:
please use `l, h` instead of `lo, hi` for consistency with other similar things in this file

Reply:
Fixed.